test/suites/rt-upgrade/test-upgrade-chain.ts

```diff
@@ -45,7 +45,7 @@ describeSuite({
       test: async () => {
         const balanceBefore = (await api.query.system.account(DUMMY_ACCOUNT)).data.free.toBigInt();
         await api.tx.balances.transfer(DUMMY_ACCOUNT, parseEther("1")).signAndSend(alith);
-        await context.createBlock();
+        await context.createBlock({ count: 2 });
```

Comment:
Why 2 blocks ?

Reply:
Because chopsticks has been randomly failing in other [CI runs](https://github.com/PureStake/moonbeam/actions/runs/5210242684/jobs/9401534547?pr=2332), which i suspect is a racecondition. I figured generating an extra block should ensure the state is properly updated.